components/tooltip/tooltip-help.js

```diff
@@ -91,11 +97,11 @@ class HelpTooltip extends FocusMixin(FocusVisiblePolyfillMixin(LitElement)) {
 			<button id="d2l-tooltip-help-text" class="${classMap(classes)}">
 				${this.text}
 			</button>
-			<d2l-tooltip for="d2l-tooltip-help-text" delay=0 offset=13>
+			<d2l-tooltip for="d2l-tooltip-help-text" delay=0 offset=13 position="${ifDefined(this.position)}">
```

Comment:
Nit: attributes in HTML are always strings, so I like to be consistent with always surrounding them in double quotes.

```suggestion
			<d2l-tooltip for="d2l-tooltip-help-text" delay="0" offset="13" position="${ifDefined(this.position)}">
```